ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/SubmitterInfo.scala

```diff
@@ -10,7 +10,12 @@ import java.time.Instant
   * Note that this is used for party-originating changes only. They are
   * usually issued via the Ledger API.
   *
-  * @param submitter: the party that submitted the change.
+  * @param actAs: the set of parties that submitted the change.
+  *
+  * @param readAs: the set of parties on whose behalf (in addition to all
```

Comment:
`readAs` is not used on the read path. Should we document that `ReadService` implementations may set this field to an empty set?

Reply:
That sounds like a good idea to me.

Reply:
Removing `readAs`, as it's not used on the ledger side.